extension.css

```diff
@@ -90,48 +79,52 @@ select.browser-style:-moz-focusring * {
   text-shadow: none;
 }
 
-button.browser-style.hover,
-select.browser-style.hover {
-  background-color: #ebebeb;
-  border: 1px solid #b1b1b1;
+button.browser-style:hover,
+select.browser-style:hover {
+  background-color: var(--grey-90-a20);
 }
 
-button.browser-style.pressed,
-select.browser-style.pressed {
-  background-color: #d4d4d4;
-  border: 1px solid #858585;
+button.browser-style:hover:active,
+select.browser-style:hover:active {
+  background-color: var(--grey-90-a30);
 }
 
-button.browser-style.disabled,
-select.browser-style.disabled {
-  color: #999;
-  opacity: .5;
+button.browser-style:disabled,
+select.browser-style:disabled {
+  opacity: .4;
 }
 
-button.browser-style.focused,
-select.browser-style.focused {
-  border-color: #fff;
-  box-shadow: 0 0 0 2px rgba(97, 181, 255, 0.75);
+button.browser-style:focus,
+select.browser-style:focus {
+  border-color: var(--blue-50);
+  box-shadow: 0 0 0 1px var(--blue-50), 0 0 0 4px var(--blue-50-a30);
 }
 
 button.browser-style.default {
-  background-color: #0996f8;
-  border-color: #0670cc;
+  -moz-osx-font-smoothing: grayscale;
+  background-color: var(--blue-60);
   color: #fff;
```

Comment:
Time to introduce a `var(--white)` to Photon's colours? 😉

Reply:
I think so, let me open a PR.

Reply:
Here we go https://github.com/FirefoxUX/design-tokens/pull/41 ☺️